reporter/src/test/kotlin/reporters/OpossumReporterTest.kt

```diff
@@ -151,8 +151,7 @@ class OpossumReporterTest : WordSpec({
 
         "create a result that contains all packages in its signals" {
             result.analyzer!!.result.packages.forAll { pkg ->
-                val id = pkg.pkg.id
-                opossumInput.signals.find { it.id == id } shouldNot beNull()
+                opossumInput.signals.find { signal -> signal.id == pkg.pkg.id } shouldNot beNull()
```

Comment:
Nit: This does more than just inlining `id` as it also introduces the (superfluous) `signal` lambda parameter.